src/main/java/FrameworksDrivers/LogInView.java

```diff
@@ -73,33 +69,41 @@ public void sendPaths(Object[] paths){
 
 
     @Override
-    public void actionPerformed(ActionEvent e) {
+    public void updatePage(Object[] info) {
+        //LoginPresenter logPresenter = new LoginPresenter();
+        ...
+    }
+
+    @Override
+    public void actionPerformed(ActionEvent evt) {
         // define action performed when button clicked
         // create presenter
```

Comment:
Yes, presenter should be created here so that you can call the use case eventually